docker_console/web/engines/base/docker.py

```diff
@@ -240,6 +240,12 @@ def get_env_file(self):
         else:
             return '--env-file=.env'
 
+    def get_net(self):
+        if not cmd_options.with_net:
```

Comment:
```suggestion
        return '--network nginx-proxy' if cmd_options.with_net else ''
```

Reply:
or with additional param:
```suggestion
        return '--network %s' % cmd_options.net_name if cmd_options.with_net else ''
```